### ISSUE TITLE
Crash to desktop (backtrace available)(chat)

### ISSUE THREAD

**Issue by Fixer-007** [minetest/minetest#5661]
In some recent builds I started to have crashes during play on 0.4.15-dev multiplayer server (Hometown).
Crashed on 2818d3f on Windows 7. Since it is in some way related to chat (crash during incoming message or opening chat), here is csm mod that I've also used during crash as potentially linked to this problem: https://github.com/cheapie/chat6

Backtrace:
```
Program received signal SIGSEGV, Segmentation fault.
0x000007fefd4665d0 in pow () from C:\Windows\system32\msvcrt.dll

Thread 74 (Thread 476.0xbd8):
#0  0x0000000076f9bd7a in ntdll!ZwWaitForSingleObject () from C:\Windows\SYSTEM32\ntdll.dll
#1  0x000007fefcfe10ac in WaitForSingleObjectEx () from C:\Windows\system32\KernelBase.dll
#2  0x00000000004a01bc in Semaphore::wait (this=0x2410e530)
    at /media/sf_Exchange/minetest/build/minetest/src/threading/semaphore.cpp:103
#3  0x0000000000746387 in UpdateThread::run (this=0x2410e4b0) at /media/sf_Exchange/minetest/build/minetest/src/util/thread.h:212
#4  0x00000000004a085b in Thread::threadProc (param=0x2410e4b0)
    at /media/sf_Exchange/minetest/build/minetest/src/threading/thread.cpp:267
#5  0x0000000076d459cd in KERNEL32!BaseThreadInitThunk () from C:\Windows\system32\kernel32.dll
#6  0x0000000076f7a561 in ntdll!RtlUserThreadStart () from C:\Windows\SYSTEM32\ntdll.dll
#7  0x0000000000000000 in ?? ()
Backtrace stopped: previous frame inner to this frame (corrupt stack?)

Thread 70 (Thread 476.0x1748):
#0  0x0000000076f9bd7a in ntdll!ZwWaitForSingleObject () from C:\Windows\SYSTEM32\ntdll.dll
#1  0x000007fefcfe10ac in WaitForSingleObjectEx () from C:\Windows\system32\KernelBase.dll
#2  0x00000000004a01bc in Semaphore::wait (this=0x2936bfb0)
    at /media/sf_Exchange/minetest/build/minetest/src/threading/semaphore.cpp:103
#3  0x0000000000746387 in UpdateThread::run (this=0x2936bf30) at /media/sf_Exchange/minetest/build/minetest/src/util/thread.h:212
#4  0x00000000004a085b in Thread::threadProc (param=0x2936bf30)
    at /media/sf_Exchange/minetest/build/minetest/src/threading/thread.cpp:267
#5  0x0000000076d459cd in KERNEL32!BaseThreadInitThunk () from C:\Windows\system32\kernel32.dll
#6  0x0000000076f7a561 in ntdll!RtlUserThreadStart () from C:\Windows\SYSTEM32\ntdll.dll
#7  0x0000000000000000 in ?? ()
Backtrace stopped: previous frame inner to this frame (corrupt stack?)

Thread 69 (Thread 476.0x11e0):
#0  0x0000000076f9bd7a in ntdll!ZwWaitForSingleObject () from C:\Windows\SYSTEM32\ntdll.dll
#1  0x000007fefc431318 in ?? () from C:\Windows\system32\mswsock.dll
#2  0x000007fefc4412d7 in WSPStartup () from C:\Windows\system32\mswsock.dll
#3  0x000007fefdbc507c in select () from C:\Windows\system32\ws2_32.dll
#4  0x000007fefdbc4ffd in select () from C:\Windows\system32\ws2_32.dll
#5  0x0000000000696c96 in UDPSocket::WaitData (this=0x2412e878, timeout_ms=50)
    at /media/sf_Exchange/minetest/build/minetest/src/socket.cpp:560
#6  0x0000000000426c2c in con::ConnectionReceiveThread::receive (this=0x2412eaf0)
    at /media/sf_Exchange/minetest/build/minetest/src/network/connection.cpp:2142
#7  0x000000000042604e in con::ConnectionReceiveThread::run (this=0x2412eaf0)
    at /media/sf_Exchange/minetest/build/minetest/src/network/connection.cpp:2065
#8  0x00000000004a085b in Thread::threadProc (param=0x2412eaf0)
    at /media/sf_Exchange/minetest/build/minetest/src/threading/thread.cpp:267
#9  0x0000000076d459cd in KERNEL32!BaseThreadInitThunk () from C:\Windows\system32\kernel32.dll
#10 0x0000000076f7a561 in ntdll!RtlUserThreadStart () from C:\Windows\SYSTEM32\ntdll.dll
#11 0x0000000000000000 in ?? ()
Backtrace stopped: previous frame inner to this frame (corrupt stack?)

Thread 68 (Thread 476.0x15bc):
#0  0x0000000076f9be0a in ntdll!ZwSetInformationThread () from C:\Windows\SYSTEM32\ntdll.dll
#1  0x0000000076d34e5a in SetThreadAffinityMask () from C:\Windows\system32\kernel32.dll
#2  0x000000006e572cc4 in createDeviceEx () from d:\Programs\_Dev\TDM-GCC-64\gdb64\bin\minetest-0.4.15-2818d3f-win64\bin\Irrlicht.dll
#3  0x1024848d48fffc92 in ?? ()
Backtrace stopped: previous frame inner to this frame (corrupt stack?)

Thread 67 (Thread 476.0x1458):
#0  0x0000000076f9bd7a in ntdll!ZwWaitForSingleObject () from C:\Windows\SYSTEM32\ntdll.dll
#1  0x000007fefcfe10ac in WaitForSingleObjectEx () from C:\Windows\system32\KernelBase.dll
#2  0x000000006b626ce0 in MMDevApiProc () from d:\Programs\_Dev\TDM-GCC-64\gdb64\bin\minetest-0.4.15-2818d3f-win64\bin\OpenAL32.dll
#3  0x000000006b61f31d in StarterFunc () from d:\Programs\_Dev\TDM-GCC-64\gdb64\bin\minetest-0.4.15-2818d3f-win64\bin\OpenAL32.dll
#4  0x0000000076d459cd in KERNEL32!BaseThreadInitThunk () from C:\Windows\system32\kernel32.dll
#5  0x0000000076f7a561 in ntdll!RtlUserThreadStart () from C:\Windows\SYSTEM32\ntdll.dll
#6  0x0000000000000000 in ?? ()
Backtrace stopped: previous frame inner to this frame (corrupt stack?)

Thread 9 (Thread 476.0x16f0):
#0  0x0000000076f9c2ea in ntdll!ZwWaitForMultipleObjects () from C:\Windows\SYSTEM32\ntdll.dll
#1  0x0000000076f6a3c7 in ntdll!TpIsTimerSet () from C:\Windows\SYSTEM32\ntdll.dll
#2  0x0000000076d459cd in KERNEL32!BaseThreadInitThunk () from C:\Windows\system32\kernel32.dll
#3  0x0000000076f7a561 in ntdll!RtlUserThreadStart () from C:\Windows\SYSTEM32\ntdll.dll
#4  0x0000000000000000 in ?? ()
Backtrace stopped: previous frame inner to this frame (corrupt stack?)

Thread 3 (Thread 476.0xc6c):
#0  0x0000000076e69e4a in USER32!SfmDxSetSwapChainStats () from C:\Windows\system32\user32.dll
#1  0x0000000076e6611e in USER32!GetMessageA () from C:\Windows\system32\user32.dll
#2  0x000000006b625a6a in MMDevApiMsgProc () from d:\Programs\_Dev\TDM-GCC-64\gdb64\bin\minetest-0.4.15-2818d3f-win64\bin\OpenAL32.dll
#3  0x0000000076d459cd in KERNEL32!BaseThreadInitThunk () from C:\Windows\system32\kernel32.dll
#4  0x0000000076f7a561 in ntdll!RtlUserThreadStart () from C:\Windows\SYSTEM32\ntdll.dll
#5  0x0000000000000000 in ?? ()
Backtrace stopped: previous frame inner to this frame (corrupt stack?)

Thread 2 (Thread 476.0x1668):
#0  0x0000000076f9bd7a in ntdll!ZwWaitForSingleObject () from C:\Windows\SYSTEM32\ntdll.dll
#1  0x000007fefcfe10ac in WaitForSingleObjectEx () from C:\Windows\system32\KernelBase.dll
#2  0x000007fee69dc018 in atigktxx!XopOpenLinkedAdapter () from C:\Windows\system32\atig6txx.dll
#3  0x000007fee69dc089 in atigktxx!XopOpenLinkedAdapter () from C:\Windows\system32\atig6txx.dll
#4  0x000007fee69dbe6f in atigktxx!XopOpenLinkedAdapter () from C:\Windows\system32\atig6txx.dll
#5  0x000007fee69e6b7f in atigktxx!XopOpenLinkedAdapter () from C:\Windows\system32\atig6txx.dll
#6  0x000007fee69e6d36 in atigktxx!XopOpenLinkedAdapter () from C:\Windows\system32\atig6txx.dll
#7  0x0000000076d459cd in KERNEL32!BaseThreadInitThunk () from C:\Windows\system32\kernel32.dll
#8  0x0000000076f7a561 in ntdll!RtlUserThreadStart () from C:\Windows\SYSTEM32\ntdll.dll
#9  0x0000000000000000 in ?? ()
Backtrace stopped: previous frame inner to this frame (corrupt stack?)

Thread 1 (Thread 476.0x15d0):
#0  0x000007fefd4665d0 in pow () from C:\Windows\system32\msvcrt.dll
#1  0x0000000000505f7f in ChatBuffer::formatChatLine (this=0x22f790, line=..., cols=158, 
    destination=std::vector of length 508, capacity 1024 = {...}) at /media/sf_Exchange/minetest/build/minetest/src/chat.cpp:339
#2  0x00000000005053c4 in ChatBuffer::addLine (this=0x22f790, name=L"", 
    text=L"[01:15:55] <\033(c@#CC0000)Laramee908\033(c@#ffffff)> 1233\002\060\015\116\363\140")
    at /media/sf_Exchange/minetest/build/minetest/src/chat.cpp:56
#3  0x000000000050754c in ChatBackend::addMessage (this=0x22f790, name=L"", 
    text=L"[01:15:55] <\033(c@#CC0000)Laramee908\033(c@#ffffff)> 1233\002\060\015\116\363\140")
    at /media/sf_Exchange/minetest/build/minetest/src/chat.cpp:692
#4  0x0000000000507856 in ChatBackend::addUnparsedMessage (this=0x22f790, 
    message=L"[01:15:55] <\033(c@#CC0000)Laramee908\033(c@#ffffff)> 1233\002\060\015\116\363\140")
    at /media/sf_Exchange/minetest/build/minetest/src/chat.cpp:717
#5  0x0000000000577036 in updateChat (client=..., dtime=0.0149999997, show_debug=true, screensize=..., show_chat=true, 
    show_profiler=0, chat_backend=..., guitext_chat=0x1c891430) at /media/sf_Exchange/minetest/build/minetest/src/game.cpp:948
#6  0x000000000058632f in Game::updateFrame (this=0x22f1a0, graph=0x22f050, stats=0x22f010, dtime=0.0149999997, cam=...)
    at /media/sf_Exchange/minetest/build/minetest/src/game.cpp:4038
#7  0x0000000000579c89 in Game::run (this=0x22f1a0) at /media/sf_Exchange/minetest/build/minetest/src/game.cpp:1659
#8  0x000000000058958c in the_game (kill=0xb9c330 <porting::g_killed>, random_input=false, input=0x517d4a0, device=0x5145320, 
    map_dir="d:\\Programs\\_Dev\\TDM-GCC-64\\gdb64\\bin\\minetest-0.4.15-2818d3f-win64\\bin\\..\\worlds\\world", playername="Fixer", 
    password="removedfromthisbacktrace", address="hometownserver.com", port=30000, error_message="", chat_backend=..., 
    reconnect_requested=0x22f8af, gamespec=..., simple_singleplayer_mode=false)
    at /media/sf_Exchange/minetest/build/minetest/src/game.cpp:4526
#9  0x00000000004027ad in ClientLauncher::run (this=0x22fc50, game_params=..., cmd_args=...)
    at /media/sf_Exchange/minetest/build/minetest/src/client/clientlauncher.cpp:260
#10 0x00000000005dc6e4 in main (argc=1, argv=0x5278830) at /media/sf_Exchange/minetest/build/minetest/src/main.cpp:230
gdb crash after that
```
Verbose output in debug.txt (last lines):
```
2017-05-15 01:15:21: INFO[Main]: ClientMap: Unloaded 11 blocks from memory, 3270 blocks in memory.
2017-05-15 01:15:22: INFO[Main]: Client: time_of_day=4881 time_speed=72 dr=252
2017-05-15 01:15:23: VERBOSE[Main]: Saving registered mod storages.
2017-05-15 01:15:24: INFO[Main]: Client: avg_rtt=0.172
2017-05-15 01:15:27: INFO[Main]: Client: time_of_day=4981 time_speed=72 dr=292
2017-05-15 01:15:28: VERBOSE[Main]: Saving registered mod storages.
2017-05-15 01:15:32: INFO[Main]: Client: time_of_day=5081 time_speed=72 dr=332
2017-05-15 01:15:34: INFO[Main]: Client: avg_rtt=0.172
2017-05-15 01:15:34: VERBOSE[Main]: Saving registered mod storages.
2017-05-15 01:15:37: INFO[Main]: Client: time_of_day=5181 time_speed=72 dr=383
2017-05-15 01:15:39: VERBOSE[Main]: Saving registered mod storages.
2017-05-15 01:15:39: INFO[Main]: Client packetcounter (20):
2017-05-15 01:15:39: INFO[Main]: cmd 2 count 0
2017-05-15 01:15:39: INFO[Main]: cmd 3 count 0
2017-05-15 01:15:39: INFO[Main]: cmd 32 count 0
2017-05-15 01:15:39: INFO[Main]: cmd 33 count 0
2017-05-15 01:15:39: INFO[Main]: cmd 34 count 0
2017-05-15 01:15:39: INFO[Main]: cmd 39 count 0
2017-05-15 01:15:39: INFO[Main]: cmd 41 count 4
2017-05-15 01:15:39: INFO[Main]: cmd 48 count 5
2017-05-15 01:15:39: INFO[Main]: cmd 49 count 0
2017-05-15 01:15:39: INFO[Main]: cmd 50 count 377
2017-05-15 01:15:39: INFO[Main]: cmd 51 count 5
2017-05-15 01:15:39: INFO[Main]: cmd 52 count 0
2017-05-15 01:15:39: INFO[Main]: cmd 58 count 0
2017-05-15 01:15:39: INFO[Main]: cmd 60 count 0
2017-05-15 01:15:39: INFO[Main]: cmd 61 count 0
2017-05-15 01:15:39: INFO[Main]: cmd 63 count 0
2017-05-15 01:15:39: INFO[Main]: cmd 65 count 0
2017-05-15 01:15:39: INFO[Main]: cmd 66 count 0
2017-05-15 01:15:39: INFO[Main]: cmd 67 count 0
2017-05-15 01:15:39: INFO[Main]: cmd 69 count 0
2017-05-15 01:15:39: INFO[Main]: cmd 70 count 0
2017-05-15 01:15:39: INFO[Main]: cmd 71 count 113
2017-05-15 01:15:39: INFO[Main]: cmd 72 count 0
2017-05-15 01:15:39: INFO[Main]: cmd 73 count 0
2017-05-15 01:15:39: INFO[Main]: cmd 74 count 0
2017-05-15 01:15:39: INFO[Main]: cmd 75 count 0
2017-05-15 01:15:39: INFO[Main]: cmd 76 count 0
2017-05-15 01:15:39: INFO[Main]: cmd 77 count 0
2017-05-15 01:15:39: INFO[Main]: cmd 78 count 0
2017-05-15 01:15:39: INFO[Main]: cmd 81 count 0
2017-05-15 01:15:39: INFO[Main]: cmd 96 count 0
2017-05-15 01:15:42: INFO[Main]: Client: time_of_day=5282 time_speed=72 dr=444
2017-05-15 01:15:44: INFO[Main]: Client: avg_rtt=0.172
2017-05-15 01:15:44: VERBOSE[Main]: Saving registered mod storages.
2017-05-15 01:15:47: INFO[Main]: ClientMap: Unloaded 4 blocks from memory, 3266 blocks in memory.
2017-05-15 01:15:48: INFO[Main]: Client: time_of_day=5383 time_speed=72 dr=505
2017-05-15 01:15:50: VERBOSE[Main]: Saving registered mod storages.
2017-05-15 01:15:52: INFO[Main]: Client: time_of_day=5483 time_speed=72 dr=575
2017-05-15 01:15:53: INFO[Main]: ClientMap: Unloaded 45 blocks from memory, 3221 blocks in memory.
2017-05-15 01:15:54: INFO[Main]: Client: avg_rtt=0.172
2017-05-15 01:15:55: VERBOSE[Main]: Saving registered mod storages.
```

**Comment by bigfoot547** [comment]
Which 3 mods @Fixer-007 ?

**Comment by Fixer-007** [comment]
chat6, inspect, oredetect

**Comment by Fixer-007** [comment]
Another crash, that coincided with explosion of very large 99+ TNT on server.

**Comment by Fixer-007** [comment]
Same crash when I pressed / (opens the chat), random, can't reproduce :/

**Comment by Fixer-007** [comment]
YAY. Caught crash under gdb, backtrace posted in first message ^^^

**Comment by ExeterDad** [comment]
@Fixer-007 We have been getting more mysterious crashes at the hometown server that I've been pulling my hair out over. If it turns out to be you and your crazy testing... you might not make it to my Christmas card list this year :)
I "think" this part of the server log is the same time as your backtrace. Not sure if you were aware the server was possibly crashing at the same time as your client?  Anyhow, the logs don't show much in my opinion.

I'm not certain how to run or build the package to get you more output. I'd be willing to do so on request if performance doesn't take too much of a hit.
```
2017-05-14 13:51:10: ACTION[Server]: Baby_Cupcake digs default:stone at (-1078,-2088,88)
2017-05-14 13:51:10: ACTION[Server]: Hurtubise405 punches object 19936: player Ambroziak968
2017-05-14 13:51:10: ACTION[Server]: Player Ambroziak968 punched by player Hurtubise405, damage 0 HP
2017-05-14 13:51:10: ACTION[Server]: redstar places node default:dirt at (985,20,-1591)
2017-05-14 13:51:10: ACTION[Server]: Hurtubise405 punches object 19936: player Ambroziak968
2017-05-14 13:51:10: ACTION[Server]: Player Ambroziak968 punched by player Hurtubise405, damage 0 HP
2017-05-14 13:51:11: ACTION[Server]: redstar places node default:dirt at (985,20,-1592)
2017-05-14 13:51:11: ACTION[Server]: Hurtubise405 punches object 19936: player Ambroziak968
2017-05-14 13:51:11: ACTION[Server]: Player Ambroziak968 punched by player Hurtubise405, damage 1 HP
2017-05-14 13:51:11: ACTION[Server]: Hurtubise405 punches object 19936: player Ambroziak968
2017-05-14 13:51:11: ACTION[Server]: Player Ambroziak968 punched by player Hurtubise405, damage 0 HP
2017-05-14 13:51:11: ACTION[Server]: Hurtubise405 punches object 19936: player Ambroziak968
2017-05-14 13:51:11: ACTION[Server]: Player Ambroziak968 punched by player Hurtubise405, damage 0 HP
2017-05-14 13:51:11: ACTION[Server]: Baby-Flower digs doors:trapdoor at (795,9,-346)
2017-05-14 13:51:12: ACTION[Server]: Hurtubise405 punches object 19936: player Ambroziak968
2017-05-14 13:51:12: ACTION[Server]: Player Ambroziak968 punched by player Hurtubise405, damage 0 HP
2017-05-14 13:51:12: [Server]: AngelAshlie12 leaves game, Playercount: 36
2017-05-14 13:51:12: ACTION[Server]: AngelAshlie12 leaves game. List of players: kittykc974 Scibilia97 CALGARO Geoge jupiter tristan Terranov Vicencio745 Fixer Junho darianbrown IMS_BULLDOGS gdb Baby-Flower jomar rose102 Flowers845 yutuit Lone_Wolf ynomrah Welters6 Hurtubise405 redstar stacy
2017-05-14 13:51:12: ACTION[Server]: 46 Cookman338 Amento639 Behrle81 Spirmen psix Ambroziak968 aaauuuyya toxit Luna-HD Baby_Cupcake KITTYKATKIT2 Gilda 
2017-05-14 13:51:12: ACTION[Server]: Baby_Cupcake digs default:stone at (-1079,-2086,86)
2017-05-14 13:51:12: ACTION[Server]: Terranov places node default:dirt at (-521,1,-399)
2017-05-14 13:51:12: ACTION[Server]: Hurtubise405 punches object 19936: player Ambroziak968
2017-05-14 13:51:12: ACTION[Server]: Player Ambroziak968 punched by player Hurtubise405, damage 1 HP
2017-05-14 13:51:13: ACTION[Server]: Hurtubise405 punches object 19936: player Ambroziak968
2017-05-14 13:51:13: ACTION[Server]: Player Ambroziak968 punched by player Hurtubise405, damage 0 HP
2017-05-14 13:51:13: ACTION[Server]: CHAT: <toxit> яяяяя нет
2017-05-14 13:51:13: ACTION[Server]: Hurtubise405 punches object 19936: player Ambroziak968
2017-05-14 13:51:13: ACTION[Server]: Player Ambroziak968 punched by player Hurtubise405, damage 0 HP
2017-05-14 13:51:13: ACTION[Server]: Hurtubise405 punches object 19936: player Ambroziak968
2017-05-14 13:51:13: ACTION[Server]: Player Ambroziak968 punched by player Hurtubise405, damage 0 HP
2017-05-14 13:51:14: ACTION[Server]: Baby-Flower digs default:wood at (796,7,-346)
2017-05-14 13:51:14: ACTION[Server]: Hurtubise405 punches object 19936: player Ambroziak968
2017-05-14 13:51:14: ACTION[Server]: Player Ambroziak968 punched by player Hurtubise405, damage 0 HP
2017-05-14 13:51:14: ACTION[Server]: Baby_Cupcake digs default:stone_with_mese at (-1080,-2086,86)
2017-05-14 13:51:15: ACTION[Server]: Server: Player with an empty name tried to connect from 84.46.239.108
2017-05-14 13:51:15: ACTION[Server]: Hurtubise405 punches object 19936: player Ambroziak968
2017-05-14 13:51:15: ACTION[Server]: Player Ambroziak968 punched by player Hurtubise405, damage 1 HP
2017-05-14 13:51:15: ACTION[Server]: redstar digs default:jungletree at (986,21,-1589)
2017-05-14 13:51:15: ACTION[Server]: Hurtubise405 punches object 19936: player Ambroziak968
2017-05-14 13:51:15: ACTION[Server]: Player Ambroziak968 punched by player Hurtubise405, damage 0 HP
2017-05-14 13:51:15: ACTION[Server]: redstar digs default:jungleleaves at (986,21,-1590)
2017-05-14 13:51:16: ACTION[Server]: CHAT: <psix> я знаю что я только там добавлен
2017-05-14 13:51:16: ACTION[Server]: redstar digs default:jungletree at (986,20,-1589)
2017-05-14 13:51:17: ACTION[Server]: redstar digs default:jungletree at (986,19,-1589)
2017-05-14 13:51:17: ACTION[Server]: redstar digs default:jungletree at (986,18,-1589)
2017-05-14 13:51:18: ACTION[Server]: jupiter takes stuff from protected chest at (762,9,-323)
2017-05-14 13:51:19: ACTION[Server]: redstar digs default:jungleleaves at (985,20,-1589)
2017-05-14 13:51:19: ACTION[Server]: CHAT: <jomar> Baby-Flower
2017-05-14 13:51:19: ACTION[Server]: redstar digs default:jungletree at (985,19,-1588)
2017-05-14 13:51:19: ACTION[Server]: CHAT: <psix> да
2017-05-14 13:51:20: ACTION[Server]: Terranov places node default:dirt at (-520,1,-399)
2017-05-14 13:51:20: ACTION[Server]: redstar digs default:jungleleaves at (984,19,-1588)
2017-05-14 13:51:21: ACTION[Server]: Baby-Flower digs doors:trapdoor_open at (795,8,-346)
2017-05-14 13:51:21: ACTION[Server]: redstar digs default:jungleleaves at (984,20,-1589)
2017-05-14 13:51:21: ACTION[Server]: Server: luismigel supplied empty password from 177.38.204.26
2017-05-14 13:51:21: ACTION[Server]: redstar digs default:jungleleaves at (984,19,-1589)
2017-05-14 13:51:21: ACTION[Server]: jupiter takes stuff from protected chest at (762,9,-323)
2017-05-14 13:51:22: ACTION[Server]: redstar digs default:jungleleaves at (984,19,-1586)
2017-05-14 13:51:22: ACTION[Server]: Baby_Cupcake digs default:stone_with_gold at (-1080,-2090,102)
2017-05-14 13:51:22: ACTION[Server]: Baby_Cupcake digs default:stone at (-1080,-2090,103)
2017-05-14 13:51:22: ACTION[Server]: redstar digs default:jungleleaves at (985,19,-1587)
2017-05-14 13:51:23: ACTION[Server]: Baby_Cupcake digs default:stone at (-1078,-2089,103)
2017-05-14 13:51:23: ACTION[Server]: redstar digs default:jungletree at (985,19,-1586)
2017-05-14 13:51:23: ACTION[Server]: Baby_Cupcake digs default:stone_with_gold at (-1079,-2090,104)
2017-05-14 13:51:23: ACTION[Server]: Terranov places node default:torch_wall at (-520,2,-399)
2017-05-14 13:51:24: ACTION[Server]: redstar digs default:jungleleaves at (984,19,-1587)
2017-05-14 13:51:24: ACTION[Server]: CHAT: <Flowers845> What happens when u eat a gold egg
2017-05-14 13:51:24: ACTION[Server]: Baby_Cupcake digs default:stone_with_gold at (-1078,-2090,103)
2017-05-14 13:51:24: ACTION[Server]: Baby_Cupcake digs default:stone at (-1078,-2090,104)
2017-05-14 13:51:24: ACTION[Server]: Baby_Cupcake digs default:stone at (-1079,-2090,103)
2017-05-14 13:51:24: ACTION[Server]: CHAT: <Behrle81> hi
2017-05-14 13:51:25: ACTION[Server]: CHAT: <Baby-Flower> Yes
2017-05-14 13:51:25: ACTION[Server]: Baby_Cupcake digs default:stone_with_iron at (-1081,-2091,103)
2017-05-14 13:51:26: ACTION[Server]: Baby_Cupcake digs default:stone at (-1080,-2091,103)
2017-05-14 13:51:26: ACTION[Server]: CHAT: <psix> опять
2017-05-14 13:51:26: ACTION[Server]: redstar digs default:jungleleaves at (986,20,-1586)
2017-05-14 13:51:26: ACTION[Server]: player jupiter crafts default:diamond 9
2017-05-14 13:51:27: ACTION[Server]: redstar digs default:jungleleaves at (987,20,-1587)
2017-05-14 13:51:27: ACTION[Server]: player jupiter crafts default:diamond 9


-------------
  Separator
-------------

2017-05-14 13:51:31: WARNING[Main]: Mod name conflict detected: "farming"
2017-05-14 13:51:31: WARNING[Main]: Will not load: /usr/share/minetest/games/minetest_game/mods/farming
2017-05-14 13:51:31: WARNING[Main]: Overridden by: /home/leeh/.minetest/worlds/server/worldmods/farming
2017-05-14 13:51:31: WARNING[Main]: Mod name conflict detected: "sethome"
2017-05-14 13:51:31: WARNING[Main]: Will not load: /usr/share/minetest/games/minetest_game/mods/sethome
2017-05-14 13:51:31: WARNING[Main]: Overridden by: /home/leeh/.minetest/worlds/server/worldmods/sethome
2017-05-14 13:51:31: WARNING[Main]: Mod name conflict detected: "stairs"
2017-05-14 13:51:31: WARNING[Main]: Will not load: /usr/share/minetest/games/minetest_game/mods/stairs
2017-05-14 13:51:31: WARNING[Main]: Overridden by: /home/leeh/.minetest/worlds/server/worldmods/stairs
2017-05-14 13:51:31: WARNING[Main]: Mod name conflict detected: "tnt"
2017-05-14 13:51:31: WARNING[Main]: Will not load: /usr/share/minetest/games/minetest_game/mods/tnt
2017-05-14 13:51:31: WARNING[Main]: Overridden by: /home/leeh/.minetest/worlds/server/worldmods/tnt
2017-05-14 13:51:31: WARNING[Main]: Your PostgreSQL server lacks UPSERT support. Use version 9.5 or better if possible.
2017-05-14 13:51:31: WARNING[Main]: NodeDefManager: Ignoring CONTENT_IGNORE redefinition
2017-05-14 13:51:32: ACTION[Main]: hb.register_hudbar: health
2017-05-14 13:51:32: ACTION[Main]: hb.register_hudbar: breath
2017-05-14 13:51:32: ACTION[Main]: [warps] loaded 
2017-05-14 13:51:32: ACTION[Main]: hb.register_hudbar: satiation
2017-05-14 13:51:32: ACTION[Main]:         .__               __                   __   
2017-05-14 13:51:32: ACTION[Main]:   _____ |__| ____   _____/  |_  ____   _______/  |_ 
2017-05-14 13:51:32: ACTION[Main]:  /     \|  |/    \_/ __ \   __\/ __ \ /  ___/\   __\
2017-05-14 13:51:32: ACTION[Main]: |  Y Y  \  |   |  \  ___/|  | \  ___/ \___ \  |  |  
2017-05-14 13:51:32: ACTION[Main]: |__|_|  /__|___|  /\___  >__|  \___  >____  > |__|  
2017-05-14 13:51:32: ACTION[Main]:       \/        \/     \/          \/     \/        
2017-05-14 13:51:32: ACTION[Main]: World at [/home/leeh/.minetest/worlds/server]
2017-05-14 13:51:32: ACTION[Main]: Server for gameid="minetest" listening on 0.0.0.0:30000.
2017-05-14 13:51:32: ACTION[Server]: Announcing to servers.minetest.net
2017-05-14 13:51:36: ACTION[Server]: darianbrown [69.10.116.77] joins game. 
2017-05-14 13:51:36: ACTION[Server]: darianbrown joins game. List of players: darianbrown
2017-05-14 13:51:36: [Server]: darianbrown joins game, Playercount: 1 Most Players: 1 Unique logins: 1
2017-05-14 13:51:37: ACTION[Server]: Server: User Luna-HD at 80.229.10.80 supplied wrong password (auth mechanism: SRP).
2017-05-14 13:51:40: ACTION[Server]: Server: User kittykc974 at 162.238.59.109 supplied wrong password (auth mechanism: SRP).
2017-05-14 13:51:42: ACTION[Server]: Terranov [181.221.143.77] joins game. 
2017-05-14 13:51:42: ACTION[Server]: Terranov joins game. List of players: darianbrown Terranov
2017-05-14 13:51:43: [Server]: Terranov joins game, Playercount: 2 Most Players: 2 Unique logins: 2
2017-05-14 13:51:47: ACTION[Server]: Cookman338 [71.93.125.83] joins game. 
2017-05-14 13:51:47: ACTION[Server]: Cookman338 joins game. List of players: darianbrown Terranov Cookman338
2017-05-14 13:51:47: [Server]: Cookman338 joins game, Playercount: 3 Most Players: 3 Unique logins: 3
```

**Comment by Fixer-007** [comment]
@ExeterDad I think it is different, btw, If possible would you search for certain chat message from player: Laramee908 at [01:15:55] GMT+3, this message probably caused this.

He posted smth like this: 12330Nó` <--- this caused my crash it seems, I wonder what is his original message, also, my client crashed but not your server, it was fine. If you have some strange minetest server crashes (not just lua errors but segmentation faults), you may probably need to compile debug one and run under gdb to get a backtrace, if that is really annoying problem.

**Comment by ExeterDad** [comment]
@Fixer-007 
Okay... it's finally worked out. You are 8 hours ahead of hometown.
The line you are looking for was found. There's a strange dot after 1233 then the rest appears to be Chinese. (Google translate > Do not want to)
I'll include a screenie of the line in case we see it differently.
```
2017-05-14 17:15:55: ACTION[Server]: Diogenes punches object -13711: LuaEntitySAO at (-105.763,22,-101)
2017-05-14 17:15:56: ACTION[Server]: stacy146 digs default:snowblock at (-1593,30,29)
2017-05-14 17:15:56: ACTION[Server]: diamondknight digs default:dirt at (-118,1,-412)
2017-05-14 17:15:56: ACTION[Server]: CU1031 right-clicks object -13683: LuaEntitySAO at (132.114,35.701,-135.966)
2017-05-14 17:15:56: ACTION[Server]: CHAT: <Laramee908> 1233。不想
2017-05-14 17:15:56: ACTION[Server]: diamondknight digs default:dirt at (-118,1,-413)
2017-05-14 17:15:56: ACTION[Server]: diamondknight digs default:dirt at (-118,1,-414)
2017-05-14 17:15:57: ACTION[Server]: CU1031 places node default:stone at (132,36,-136)
2017-05-14 17:15:57: ACTION[Server]: diamondknight digs default:dirt at (-118,1,-415)
2017-05-14 17:15:57: ACTION[Server]: Night_Wolf places node default:torch_wall at (-528,-366,-324)
```
![logline](https://cloud.githubusercontent.com/assets/9083807/26076210/f3d81e7c-3985-11e7-90fc-5c6f41f8a060.png)

**Comment by Fixer-007** [comment]
@red-001 Why don't you make a PR with your fix, I'm still testing it though.

**Comment by red-001** [comment]
@Fixer-007 #5775

**Comment by Fixer-007** [comment]
Thanks

**Comment by paramat** [comment]
@Fixer-007 was this fixed by #5775 ?

**Comment by Fixer-007** [comment]
I think so, I've played noticeable time since patch, there were no crashes.
